sx-auth.el

```diff
@@ -80,11 +81,12 @@ Methods are of the form \(METHOD . SUBMETHODS) where SUBMETHODS
 If all SUBMETHODS require auth or there are no submethods, form
 will be \(METHOD . t)")
 
-(defvar sx-auth-filter-auth '(question.upvoted
-                              question.downvoted
-                              answer.upvoted
-                              answer.downvoted
-                              comment.upvoted)
+(defconst sx-auth-filter-auth
+  '(question.upvoted
+    question.downvoted
+    answer.upvoted
+    answer.downvoted
+    comment.upvoted)
```

Comment:
Note to self: abstract out the call to the tree splicer and provide it as a separate macro/function.